merge/nested_test.go

```diff
@@ -78,23 +78,23 @@ var nestedTypeParser = func() typed.ParseableType {
     keys:
     - name
 - name: map
-  map:
-    elementType:
+  struct:
```

Comment:
should we find a better name than struct now?

Reply:
map might be the better name